scripts/multinode_distributions.sh

```diff
@@ -0,0 +1,127 @@
+#/bin/sh
+
+display_usage() {
+    printf "** Please check the exported values:: **\n Daemon : $DAEMON\n Denom : $DENOM\n ChainID : $CHAINID\n Daemon home : $DAEMON_HOME\n"
+    exit 1
+}
+
+if [ -z $DAEMON ] || [ -z $DENOM ] || [ -z $CHAINID ] || [ -z $DAEMON_HOME]
+then 
+    display_usage
+fi
+
+echo
+
+# read no.of nodes
+NODES=$1
+NODES=$1
+if [ -z $NODES ]
+then
+    NODES=5
+fi
+
+echo "** Number of nodes mentioned : $NODES **"
+
+IP="$(dig +short myip.opendns.com @resolver1.opendns.com)"
```

Comment:
If for some reason this command does not work we should take `127.0.0.1` as default